Tribler/Core/bootstrap.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import
+
+import logging
+from binascii import hexlify, unhexlify
+
+from Tribler.Core.DownloadConfig import DownloadStartupConfig
+from Tribler.Core.TorrentDef import TorrentDef, TorrentDefNoMetainfo
+
+
+class Bootstrap(object):
```

Comment:
A few lines explaining the purpose of this class would be helpful for new developers 👍

Reply:
Done